lib/omniauth/strategies/openid_connect.rb

```diff
@@ -173,6 +184,11 @@ def authorize_uri
 
         opts.merge!(options.extra_authorize_params) unless options.extra_authorize_params.empty?
 
+        if options.pkce
+          opts.merge!(pkce_authorize_params)
+          session["omniauth.pkce.verifier"] = options.pkce_verifier
```

Comment:
There are multiple code style offenses. Please run `rubocop .` to see what could be improved. Other than this little thing PR seems to be OK for me 👍

Reply:
Multiple offenses fixed, the only one that was a little tricky was class length, I could add an exception for it?

Reply:
No worries, let's keep it this way. Maybe we'll look into splitting it in future.